image/fetcher_test.go

```diff
@@ -51,7 +52,7 @@ func testFetcher(t *testing.T, when spec.G, it spec.S) {
 	it.Before(func() {
 		repo = "some-org/" + h.RandString(10)
 		repoName = registryConfig.RepoName(repo)
-		fetcher = image.NewFetcher(mocks.NewMockLogger(ioutil.Discard), docker)
+		fetcher = image.NewFetcher(fakes.NewFakeLogger(ioutil.Discard), docker)
```

Comment:
Why do we create fake loggers for these tests instead of creating real loggers that log to `ioutil.Discard`?

Reply:
As discussed, not worrying about it at this time given it was pre-existing.